.gitlab-ci.yml

```diff
@@ -2,11 +2,9 @@ include:
   - remote: https://raw.githubusercontent.com/ika-rwth-aachen/docker-ros/$DOCKER_ROS_GIT_REF/.gitlab-ci/docker-ros.yml
 
 variables:
-  BASE_IMAGE: ubuntu:20.04
-  ROS_DISTRO: noetic
-  COMMAND: rostopic list
+  BASE_IMAGE: rwthika/ros2:jazzy
```

Comment:
Why switch away from Ubuntu? This was to test that ROS is installed if missing.

Reply:
We used the ubuntu base image only for ROS 1. We could add this also for ROS 2, but better in a different PR?